icann-rdap-client/src/test_files/nic.xyz.json

```diff
@@ -0,0 +1,347 @@
+        "URL of the ICANN RDDS Inaccuracy Complaint Form: https://www.icann.org/wicf/"
+      ],
+      "links": [
+        {
+          "href": "https://www.icann.org/wicf/",
+          "title": "ICANN RDDS Inaccuracy Complaint Form"
+        }
+      ]
+    },
+    {
+      "title": "Data Policy",
+      "description": [
+        "The contact information for one or more entities associated with this domain has been redacted in accordance with the entity's disclosure preferences."
+      ]
+    }
+  ],
+  "objectClassName": "domain",
+  "handle": "D1465615-CNIC",
+  "links": [
+    {
+      "rel": "self",
+      "href": "https://rdap.centralnic.com/xyz/domain/nic.xyz",
+      "title": "Authoritative URL for this resource",
+      "type": "application/rdap+json"
+    },
+    {
+      "rel": "help",
+      "href": "https://whois.nic.xyz/rdap",
+      "title": "RDAP Service Help"
+    },
+    {
+      "href": "https://gen.xyz/",
+      "title": "XYZ.com, LLC"
+    },
+    {
+      "rel": "related",
+      "href": "https://rdap.centralnic.com/xyz/domain/nic.xyz",
+      "title": "URL of Sponsoring Registrar's RDAP Record",
+      "type": "application/rdap+json"
+    }
+  ],
+  "events": [
+    {
+      "eventAction": "registration",
+      "eventDate": "2013-09-11T11:58:06.0Z"
+    },
+    {
+      "eventAction": "expiration",
+      "eventDate": "2024-09-11T23:59:59.0Z"
+    },
+    {
+      "eventAction": "last update of RDAP database",
+      "eventDate": "2024-06-12T13:31:47.0Z"
+    },
+    {
+      "eventAction": "last changed",
+      "eventDate": "2023-09-05T00:05:10.0Z"
+    }
+  ],
+  "status": [
+    "server transfer prohibited",
+    "server update prohibited",
+    "server delete prohibited",
+    "server renew prohibited"
+  ],
+  "port43": "whois.nic.xyz",
+  "entities": [
+    {
+      "objectClassName": "entity",
+      "remarks": [
+        {
+          "title": "Data Policy",
+          "description": [
+            "Some of the data in this object has been removed."
+          ]
+        },
+        {
+          "title": "REDACTED FOR PRIVACY",
+          "description": [
+            "Some of the data in this object has been removed."
+          ]
+        },
+        {
+          "title": "EMAIL REDACTED FOR PRIVACY",
+          "description": [
+            "Please query the RDDS service of the Registrar of Record identified in this output for information on how to contact the Registrant of the queried domain name."
+          ]
+        }
+      ],
+      "status": [
+        "removed"
+      ],
+      "vcardArray": [
+        "vcard",
+        [
+          [
+            "version",
+            {},
+            "text",
+            "4.0"
+          ],
+          [
+            "kind",
+            {},
+            "text",
+            "org"
+          ],
+          [
+            "org",
+            {},
+            "text",
+            "XYZ.COM LLC"
+          ],
+          [
+            "adr",
+            {},
```

Comment:
to represent the country code, this JSON object should be `"cc":"US"`.
See https://anewton1998.github.io/rdap_guide/protocol/jcard_and_vcard.html#adr-property